docker-compose-prod.yml

```diff
@@ -18,13 +18,19 @@ services:
       - database
 
   database:
-    image: postgres:13
+    image: mysql:5.7
```

Comment:
is there any reason we are still need to stick with older version? I know our other app still are on 5.7 when 8 is available. We can talk to DBA team if they support 8 or not? does LTIJS support latest version of MySQL as well?

Reply:
I would also fix the version to be specific to what we're likely running. Last I checked it was mysql:5.7.26 but might have updated to 5.7.33. There are some minor issues that have arisen from using a different database locally verses in production.

I'm also thinking we should change the flag of 
`    restart: on-failure`

in all of our apps to either `unless-stopped` or just `off` so it doesn't restart all of the MySQL containers after a restart. It's a little annoying having to kill them off, maybe I'm the only one.

Reply:
Last time I checked they only supported MySQL 5.7. There's some general debate among MySQL users on whether to update to MySQL 8 or just switch to MariaDB. (Which is more generally more compatible with 5.7 and has different features)

I'd just use whatever MiDatabase supports but we'd want to wait till they update and we could check.

Reply:
OK, we will stick with 5.7. See conversation in the TLDev slack channel if you want.

Reply:
@jonespm

Regarding the `restart` policy, I think I'll just remove it for now (default is `off`). It doesn't seem super important, and the `on-failure` behavior could get annoying (though I don't know if I've ever experienced it myself).

How much of a risk is it to stick just with `mysql:5.7`?  We always test with MiDatabase instances before we roll to production. I feel like the time spent debugging problems with patch differences (which should in theory be no time) could be less than keeping up with each patch change the MiDatabase team rolls out.

Reply:
Regarding the patch version question, I'm going to merge this as is, and when/if we decide to do this for other applications, we can do it here as well.